Add team members to the Community page

josephinehall opened the pull request:
Hello! This PR gets a basic implementation for showing team members on the Community page:

- Adds a data model for core team members
- Adds example markdown files for team member bios
- Adds a loader to read, parse and insert the the team member content into the DB
- Shows the team members on the Community page while we figure out the site map
- Links through to websites, GH profiles and shows the team members' GH avatars

It doesn't handle inactive team members at the moment; I can add that behaviour next if you like?

A couple of the commits have a different user as the committer on them; I didn't have my gitconfig set up properly on my other computer, apologies.

A lil screenie for ease of reviewing:

<img width="744" height="756" alt="Screenshot 2025-09-03 at 8 49 17 pm" src="https://github.com/user-attachments/assets/2d315078-1b4f-4efd-94b2-46dc0837496c" />

timriley reviewed (commented):
This is looking great, @josephinehall! You've got the hang of things very quickly :)

I like that you chose individual files for the team members, this makes them nice and straightforward to edit on the filesystem.

**On the schema:**

- I reckon we can drop the description/bio text for now, in the spirit of keeping this simple and easy to populate. We can always choose to add it later.
- Let's add a `url` attribute, which can be whatever URL the person wants their name to link to. This attribute can be optional.
- Let's also add a `github_user` attribute. This should be required. We can use this to display someone's avatar image. GitHub gives us stable URLs for this, in `https://github.com/[username].png?size=[pixels]` format.
- Let's also add a `team` attribute. For now, this can have values that are one of "core" and "maintainers".

**On the data:**

Thanks for getting started with populating the people! I'll give you a full list you can use now:

```
Tim Riley
team: core
location: Australia
url: https://timriley.info
github_user: timriley
active_since: 2015

Sean Collins
team: core
location: USA
url: https://seancollins.tech
github_user: cllns
active_since: 2015

Nikita Shilnikov
team: core
location: Spain
github_user: flash-gordon
active_since: 2015

Max Wheeler
team: maintainers
location: Australia
github_user: makenosound
active_since: 2025

Aaron Moodie
team: maintainers
location: Australia
github_user: aaronmoodie
active_since: 2024

Sven Schwyn
team: maintainers
location: Europe
github_user: svoop
active_since: 2025

Marc Busqué
team: maintainers
location: Spain
github_user: waiting-for-dev
active_since: 2023

Adam Lassek
team: maintainers
location: USA
github_user: alassek
active_since: 2024

Kyle Plump
team: maintainers
location: USA
github_user: kyleplump
active_since: 2024

Krzysztof Piotrowski
team: maintainers
location: Poland
github_user: krzykamil
active_since: 2024

Damian C. Rossney
team: maintainers
location: USA
github_user: dcr8898
active_since: 2025
```

In terms of displaying these peeps on the page, we'll want a section for the core team first, followed by the maintainers. Each section should have its own title. (These will eventually become more important distinctions when we publish fuller guidelines about the contributor pathway and governance, etc.).

While you're tweaking the display of people, would you mind showing their avatars based on their GitHub usernames, too?

**Future changes:**

I don't think any of these should apply now, but I wanted to share this with you for future consideration:

- I'd like to get historical members listed too, so we can have a section acknowledging our alumni. I'd like to make sure we recognise Peter and Luca's contributions to these projects. We could probably handle this with an `active_until` attribute (if it's present, than we know they're no longer active). I don't have a full list of alumni yet, so I'd rather we wait until we can do it all at once.
- Actually, that's really about it :)

josephinehall commented:
Hey again Tim, I think this is looking better now. Obviously it's completely unstyled for the moment, and I've removed the markdown -> HTML conversion I'd put in for the descriptions. I've updated the screenshot above so you can see the general vibe.

One thing I haven't done is add any tests... not sure if a feature test is useful here, but I could add something to check the behaviour of the loader? LMK what you think.
